lib/convection/model/template/resource_property/aws_ec2_block_store_block_device.rb

```diff
@@ -0,0 +1,22 @@
+require_relative '../resource_property'
+
+module Convection
+  module Model
+    class Template
+      class ResourceProperty
+        # Represents an {http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-blockdev-template.html
+        # EC2 Block Store Block Device Property Type}
+        class EC2BlockStoreBlockDevice < ResourceProperty
+          property :delete_on_termination, 'DeleteOnTermination'
+          property :encrypted, 'Encrypted'
+          property :iops, 'Iops'
+          property :snapshot, 'SnapshotId'
+          property :volume_size, 'VolumeSize'
+          alias size volume_size
+          property :volume_type, 'VolumeType'
+          alias type volume_type
```

Comment:
Are the `alias size` and `property :volume_type` lines supposed to be duplicated here?